Specs/Scene/PrimitiveCollectionSpec.js

```diff
@@ -2,68 +2,67 @@
 defineSuite([
         'Scene/PrimitiveCollection',
         'Core/Cartesian3',
+        'Core/ColorGeometryInstanceAttribute',
         'Core/defaultValue',
+        'Core/defined',
         'Core/Ellipsoid',
+        'Core/GeometryInstance',
         'Core/Math',
-        'Renderer/ClearCommand',
+        'Core/Matrix4',
+        'Core/Rectangle',
+        'Core/RectangleGeometry',
         'Scene/HorizontalOrigin',
         'Scene/LabelCollection',
-        'Scene/Polygon',
+        'Scene/PerInstanceColorAppearance',
+        'Scene/Primitive',
         'Scene/VerticalOrigin',
-        'Specs/createCamera',
-        'Specs/createContext',
-        'Specs/createFrameState',
-        'Specs/pick',
-        'Specs/render'
+        'Specs/createScene'
     ], function(
         PrimitiveCollection,
         Cartesian3,
+        ColorGeometryInstanceAttribute,
         defaultValue,
+        defined,
         Ellipsoid,
+        GeometryInstance,
         CesiumMath,
-        ClearCommand,
+        Matrix4,
+        Rectangle,
+        RectangleGeometry,
         HorizontalOrigin,
         LabelCollection,
-        Polygon,
+        PerInstanceColorAppearance,
+        Primitive,
         VerticalOrigin,
-        createCamera,
-        createContext,
-        createFrameState,
-        pick,
-        render) {
+        createScene) {
     "use strict";
     /*global jasmine,describe,xdescribe,it,xit,expect,beforeEach,afterEach,beforeAll,afterAll,spyOn*/
 
+    var scene;
     var context;
+    var ellipsoid;
+    var rectangle;
     var primitives;
-    var frameState;
-    var us;
-    var camera;
 
     beforeAll(function() {
-        context = createContext();
-        frameState = createFrameState();
+        scene = createScene();
+        scene.primitives.destroyPrimitives = false;
+        context = scene.context;
+        ellipsoid = Ellipsoid.WGS84;
```

Comment:
You should be able to remove `ellipsoid` throughout this file since `WGS84` is the default.